package.json

```diff
@@ -37,6 +37,17 @@
                     "when": "editorTextFocus"
                 }
             ]
+        },
+        "configuration": {
+            "title": "Local History",
+            "properties": {
+                "local-history.maxEntriesPerFile": {
+                    "type": "number",
+                    "default": 10,
+                    "scope": "resource",
```

Comment:
Can you clarify why we want a `resource` scope?

Reply:
I was using a template from an extension, so I kept it with it, I'll remove it when I push the changes